src/adservice/Dockerfile

```diff
@@ -11,9 +11,14 @@ COPY . .
 RUN chmod +x gradlew
 RUN ./gradlew installDist
 
-FROM openjdk:8-alpine
-
-RUN apk add --no-cache libc6-compat
+FROM openjdk:8-slim
+
+# Download Stackdriver Profiler Java agent
+RUN apt-get update
+RUN apt-get install -qq wget
+RUN mkdir -p /opt/cprof && \
```

Comment:
please remove the .tar.gz file after it's used in the same `RUN` because it doesn't need to be present in the image anymore after the pipe is complete.

```
RUN wget | tar && \
        rm -rf x.tar.gz
```

Reply:
done. 
With both suggested changes, image size came down to 330MB.